README.md

```diff
@@ -135,6 +135,49 @@ AnyCable, k6, WebSockets, and Yabeda](https://evilmartians.com/chronicles/real-t
 - [k6 Extensions](https://grafana.com/docs/k6/latest/extensions/)
 - [GitHub Topic: xk6](https://github.com/topics/xk6) - Explore k6 extensions tagged with the xk6 label.
 
+### Official
```

Comment:
I haven't found [in the documentation](https://grafana.com/docs/k6/latest/extensions/) where "official" extensions are defined. What does it mean? maybe "Grafana supported" is a better name?

Reply:
In document: "Product DNA: k6 Extensions (Registry/Tiers/Cloud)":
> Extensions owned, maintained, and designated by Grafana as “official”.

Reply:
This document is private to k6 team, therefore the definition is not public.

Reply:
> This document is private to k6 team, therefore the definition is not public.

Then this may be a documentation problem, the term "official" still exists, but it is not publicly documented.
BTW I don't think anyone should have to explain what "official" means in this context. From this point of view, "Grafana supported" is not better either, because it is not found in the documentation either.

Reply:
> I haven't found [in the documentation](https://grafana.com/docs/k6/latest/extensions/) where "official" extensions are defined. What does it mean? maybe "Grafana supported" is a better name?

@pablochacin, I double-checked the documentation and found the definition of the "official" extension:

https://grafana.com/docs/k6/latest/extensions/explanations/extensions-registry/#extension-tiers

> Official extensions are those owned and maintained by Grafana Labs. They will have official documentation and have support provided by members of the Grafana organization.

Reply:
Shall make this heading a link to this doc then?

Reply:
Only **Extension Tiers** can be linked, **Official Extensions** and **Comnmunity Extensions** are only a bold `<li>` elements, they cannot be linked.
Do you think both headings should point to **Extension Tiers**?
Or I can copy the definitions.
BTW I don't think it's necessary to explain the terms official and community in this context...